content/api/motor-board.md

```diff
@@ -41,19 +41,19 @@ r.motor_board.m1
 ```
 
 {{% notice warning %}}
-Setting an incorrect value will raise an exception and your robot will crash.
-{{% /notice %}}  
+Setting a value outside of the range `-1` to `1` will raise an exception and your code will crash.
+{{% /notice %}}
 
-### Special Values
+### Special values
 
-In addition to the numeric values, there are two text constants that can be used. `robot.BRAKE` and `robot.COAST`. 
+In addition to the numeric values, there are two special constants that can be used: `robot.BRAKE` and `robot.COAST`.
```

Comment:
Another point which is probably for another PR: I don't like that we use `robot.BRAKE` spelling in text and then use it unqualified in the code example (`r.motor_board.m0 = BRAKE`). I'm not sure what the fix is.

Reply:
One fix would be to use an enum for this rather than constants. (We can keep the constants for backwards compatibility and just drop them from the docs). `PinMode` shows well how this can work and be documented.

Reply:
I agree that our usage of `robot.BRAKE` vs `BRAKE` appears inconsistent. There's also potential for confusion amongst newcomers between the class `Robot` and the module `robot` - especially since all our examples import names from the module (`from robot import X`) rather than importing it directly (`import robot`).

For now I think ensuring we demonstrate `from robot import BRAKE, COAST` and use the unqualified form `BRAKE` exclusively should be sufficient. I don't think an enum would be necessary here, in part because I'm not sure how one would name and describe it in a way that wouldn't also lend itself to including other motor modes (i.e. not brake or coast).

Reply:
6946af40a1a0d2e59ed791dd178bd7b74d89e60b